client/components/tinymce/plugins/contact-form/dialog/field-edit-button.jsx

```diff
@@ -0,0 +1,55 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import PureRenderMixin from 'react-pure-render/mixin';
+import classNames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import Gridicon from 'components/gridicon';
+import Popover from 'components/popover';
+
+export default React.createClass( {
```

Comment:
We can write this as either a stateless functional component, or an es6 class that extends PureComponent.

```javascript
function ContactFormDialogFieldEditButton( { expanded } ) {
// returns the same jsx as what's in the current render() method
}
```

```javascript
export default class ContactFormDialogFieldEditButton extends PureComponent {
```